compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/LargeMethodSplitter.java

```diff
@@ -136,6 +164,12 @@ private BLangFunction splitInitFunction(BLangPackage packageNode, SymbolEnv env)
                 if (newFuncBody.stmts.size() + chunkStmts.size() > methodSize) {
                     generatedFunctions.add(newFunc);
                     newFunc = createIntermediateInitFunction(packageNode, env);
+                    splitFuncCount++;
+                    if (splitFuncCount % INIT_FUNC_COUNT_PER_CLASS == 0) {
```

Comment:
Shall we check why these lines not covered?

Reply:
Removed the lines not getting covered.